sql queries/create_tables.sql

```diff
@@ -4,14 +4,21 @@ This file contains sql queries to create all the tables or views used in this pr
 
 /*Table to store film information*/
 CREATE TABLE IF NOT EXISTS W2W_Films (
-    film_key text
+    film_key int
 	,title text
     ,year int
     ,description text
+	,duration text
```

Comment:
Do we want this to be text or parse it and store it as minutes as a float?

Reply:
Sometimes it's hours and minutes, but sometimes it's number of seasons. I figured we could store as text but parse it if we ever needed to use it.